builds/ci/dotnet.yaml

```diff
@@ -61,6 +60,25 @@ jobs:
           IotDevice3ConnStr2: $(IotDevice3ConnStr2)
           IotHubConnStr2: $(IotHubConnStr2)
           IotHubMqttHeadCert: $(IotHubMqttHeadCert)
+      - script: |
+          # download kind binary and make it executable
+          curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v0.5.1/kind-$(uname)-amd64
+          chmod +x ./kind
+          # move it to $AGENT_TOOLSDIRECTORY\kind
+          mkdir $AGENT_TOOLSDIRECTORY/kind 
+          mv ./kind $AGENT_TOOLSDIRECTORY/kind/kind
+          # make kind to be available for integration tests 
+          echo "##vso[task.setvariable variable=PATH;]$AGENT_TOOLSDIRECTORY/kind:$PATH"
+        displayName: Install Kubernetes cluster manager (kind)
+      - script: |
+          kind create cluster --name "edgy"
+          KUBECONFIG=$(kind get kubeconfig-path --name "edgy")
+          echo "##vso[task.setvariable variable=KUBECONFIG;]$KUBECONFIG"
+          echo "##vso[task.setvariable variable=USE_EXISTING_KUBERNETES_CLUSTER;]true"
+        displayName: Create Kubernetes cluster (edgy)
+      - script: |
+          sudo -E bash -c './scripts/linux/runTests.sh "--filter $(test.filter)&Category=Kubernetes" $(Build.Configuration)'
+        displayName: Kubernetes Integration Tests
```

Comment:
It's fine to run the Kubernetes integration tests separately from the others, but why?

Reply:
It takes much more time to finish tests. And I'd like to actually know the number. Potentially it could run as separate job

Reply:
Makes sense. I was just curious